src/main/java/org/spongepowered/api/command/conversation/ConversationManager.java

```diff
@@ -0,0 +1,127 @@
+/*
+ * This file is part of SpongeAPI, licensed under the MIT License (MIT).
+ *
+ * Copyright (c) SpongePowered <https://www.spongepowered.org>
+ * Copyright (c) contributors
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.spongepowered.api.command.conversation;
+
+import org.spongepowered.api.event.cause.Cause;
+import org.spongepowered.api.event.cause.conversation.ConversationEndType;
+import org.spongepowered.api.event.command.TabCompleteEvent;
+import org.spongepowered.api.plugin.PluginContainer;
+import org.spongepowered.api.world.Location;
+import org.spongepowered.api.world.World;
+
+import java.util.Collection;
+import java.util.List;
+import java.util.Optional;
+
+import javax.annotation.Nullable;
+
+/**
+ * Handles all active conversations. This is where conversant input is processed
+ * as well as where conversations are started by Sponge. There are some methods
+ * that can be used for easy retrieval of conversations as well.
+ */
+public interface ConversationManager {
+
+    /**
+     * Gets an active {@link Conversation} with the specified id and creator.
+     *
+     * @param plugin The plugin who created the conversation
+     * @param id The id of the conversation
+     * @return The matching conversation, if present
+     */
+    Collection<Conversation> getConversation(PluginContainer plugin, String id);
```

Comment:
But the idea is that it returns either one conversation, with the given id, or nothing. Having the collection implies returning multiple conversations is supported, however multiple conversations under the same plugin and id should not exist.